### PR TITLE
adding "accessing config" FAQ answer

### PR DESCRIPTION
**Pull request by grawk**

### CODE REVIEW

File: _includes/docs/gettingstarted.md [hunk 1]
```diff
@@ -349,6 +349,18 @@ This error likely indicates that you have not built your project. Try running `g
...
 
+#### How should I access configuration values in my application?
+
+Since kraken compiles your configuration at runtime -- taking in to account the environment plus shortstop handlers -- you should not attempt to `require` the JSON directly.
+
+You can access the configuration from the request: `req.app.kraken.get("my:config:property")`.
+
+You can also access the configuration in an `onconfig` handler as discussed here: https://github.com/krakenjs/kraken-js#options
+
+If you are using an onconfig handler, just be aware that it's asynchronous. I.e. don't rely on it until onconfig has fired, or the app emits the start event.
```
Comment (totherik) [inline _includes/docs/gettingstarted.md]: I feel like `it's` and `it` are dangling pronouns. What do they refer to here? The handler in the first sentence and the config object itself in the second?

Comment (grawk) [inline _includes/docs/gettingstarted.md]: Do you think that clears it up well enough?

**Comment by totherik** [comment]
:+1:

**Comment by pvenkatakrishnan** [comment]
:+1:

**Comment by grawk** [comment]
Thanks!

**Pull request merged**